### PR TITLE
Update Tago integration guide

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Updating the integration guide for TagoIO. 

#### Changes
Instead of using a custom HTTP device, a custom TTN device is now used. Instead of granting access via device token, authentication parameter is now used. Webhook URL is also changed accordingly.

#### Testing
Tested with TTS running locally.

#### Notes for Reviewers
Changes were made after a review from Tago.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/webhooks/cloud-integrations/tagoio/tagoio-setup/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "TagoIO Setup"
+title: "Tago Setup"
```
Comment (adriansmares) [inline doc/content/integrations/webhooks/cloud-integrations/tagoio/tagoio-setup/_index.md]: I still see their name as `TagoIO` on their website. Is this change intentional ?

File: doc/content/integrations/webhooks/cloud-integrations/tagoio/tts-setup/_index.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ Next, create a Webhook integration on {{% tts %}} by following this section.
 
 Fill in the **Webhook ID** field. 
 
-Create a `Content-Type` header entry with `application/json` value, and a `Device-Token` header entry with the token value you previously copied from TagoIO.
+Create an `Authorization` header entry with the authorization copied in the previous step as a value.
```
Comment (adriansmares) [inline doc/content/integrations/webhooks/cloud-integrations/tagoio/tts-setup/_index.md:17]: Shouldn't we drop this completely and point this to the template ?

Comment (nejraselimovic) [reply doc/content/integrations/webhooks/cloud-integrations/tagoio/tts-setup/_index.md:17]: Sure, I can do it, it makes sense now when we have webhook templates set up. @laurensslats can you confirm if I should update all the integration guides to point to webhook templates now and remove this kind of info that's basically hidden from the user during creation?

Comment (nejraselimovic) [reply doc/content/integrations/webhooks/cloud-integrations/tagoio/tts-setup/_index.md:17]: @adriansmares just for the record, we have decided to keep this, but to mention that there is a webhook template available in the beginning of the doc

**Review by laurensslats — APPROVED** [review]
